tests/ctst/steps/azureArchive.ts

```diff
@@ -372,10 +406,14 @@ When('i run sorbetctl to retry failed restore for {string} location', async func
         --kafka-object-task-topic=${this.parameters.KafkaObjectTaskTopic} \
         --kafka-brokers ${this.parameters.KafkaHosts}`;
     try {
-        await util.promisify(exec)(command);
+        process.stdout.write(`Running command: ${command}\n`);
+        const result = await util.promisify(exec)(command);
+        process.stdout.write(`Sorbetctl command result: ${result.stdout}\n`);
     } catch (err) {
         assert.ifError(err);
     }
+    // Wait for backbeat to process the retry and update the object MDs
+    await Utils.sleep(60000);
 });
 
```

Comment:
can we not use the metadata `x-amz-meta-scal-s3-restore-attempt` to wait for backebat ?
(I'm not sure if it's visible via a headObject)

Reply:
isn't that what we are doing here ?

Reply:
You are right, MB..

That said, isnt this MD deleted when the restore succeed ?

Reply:
I think so yes, but that's not an issue here.
Here we add a delay to make sure backbeat processed the object, we can do the same by waiting for the object to have `x-amz-meta-scal-s3-restore-attempt: 1` in its metadata.

Reply:
I think it's actually an issue, when we retry, the restore succeed then the is deleted ? Or am i missing something ?

Reply:
The restore is triggered only once the object metadata got updated with `x-amz-meta-scal-s3-restore-attempt`.
Maybe i understood you comment wrong, are we waiting for the restore to be fully done ? or just for backbeat to process the initial retry order (so before sorbet restores the object) ?

Reply:
Here we are retrying for real, so there maybe a little time where `x-amz-meta-scal-s3-restore-attempt` is set and right after the MDs are updated, so i dont think we can rely on that..
Here the failure is not a real failure, it's just that we are not notified of the success of the restore (since it is azurite the restore is done instantly), so the retry is really quick as well..